app/renderer/js/notification/helpers.js

```diff
@@ -3,6 +3,31 @@ const { remote } = require('electron');
 // Do not change this
 const appId = 'org.zulip.zulip-electron';
 
+const botsList = [];
+let botsListLoaded = false;
+
+function loadBots(sync = false) {
```

Comment:
@abhigyank would be great if you can add few comments here about the function?  In general, it's better to document your code so that other devs can see  what's going on :)